ui/panels-plugin/src/plugins/time-series-chart/TimeSeriesChartPanel.tsx

```diff
@@ -54,7 +54,7 @@ export function TimeSeriesChartPanel(props: TimeSeriesChartProps) {
   } = props;
   const chartsTheme = useChartsTheme();
   const muiTheme = useTheme();
-  const echartsPalette = chartsTheme.echartsTheme.color ?? [muiTheme.palette.primary];
+  const echartsPalette = chartsTheme.echartsTheme.color;
```

Comment:
What if we called this variable `categoricalPalette`? Do you think that would be more clear / does that accurately represent this?